api/src/client/ApiFormationHelpers.ts

```diff
@@ -212,15 +221,17 @@ export const makePostBody = (
           formationFormData.businessStartDate,
           defaultDateFormat
         ).format(formationApiDateFormat),
-        MainAddress: {
-          Address1: formationFormData.addressLine1,
-          Address2: formationFormData.addressLine2,
-          City: formationFormData.addressMunicipality?.name ?? formationFormData.addressCity ?? "",
-          State: formationFormData.addressState?.name,
-          Province: formationFormData.addressProvince,
-          Zipcode: formationFormData.addressZipCode,
-          Country: formationFormData.addressCountry,
-        },
+        MainAddress: isStartingBusinessAddressPartiallyEmpty()
```

Comment:
[pebble] I'd like this function to be something more generic, and the function itself should clearly state the logic we intend to support inside.

```suggestion
        MainAddress: getBusinessAddress()
```